src/definition/messages/MessageProcessingType.ts

```diff
@@ -0,0 +1,4 @@
+export enum MessageProcessingType {
+    SendMessage = 'sendMessage',
+    RespondWithMessage = 'respondWithMessage',
```

Comment:
Wish there was documentation added that said what each of these do. They both seem the same to me :(

Does `sendMessage` actually send a message without populating the text area for the user to edit beforehand? Or is that `respondWithMessage` or what?